City-of-Helsinki/drupal-infofinland
# UHF-X delete revision

[Related to ticket IFU-938](https://helsinkisolutionoffice.atlassian.net/browse/IFU-938)

Updated helfi-drupal modules. Api_base update includes the proper functionality to remove excess revisions. The functionality is run via drush command mentioned in testing instructions.

## What was done
- Updated helfi_api_base: 
  - Added health check module
  - disabled old revision delete module
  - disabled filelog module

Unrelated: 
- Added git PR template
- Updated helfi_azure_fs for D10

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-X_delete_revision`
  * `make fresh`
* Run `make drush-cr`

## How to test
- You preferabyly should find a node with more than 6 revisions (mentioned on IFU-938 ticket)
- Inside container, run `drush helfi:revision:delete node`
- Inside container, run `drush helfi:revision:delete paragraph`
- Go check out the content you found earlier
  - All translation should have maximum of 5 revisions or (5 + current revision)
